client/state/posts/test/reducer.js

```diff
@@ -366,4 +370,105 @@ describe( 'reducer', () => {
 			expect( state ).to.eql( {} );
 		} );
 	} );
+
+	describe( '#siteRequests()', () => {
+		it( 'should default to an empty object', () => {
+			const state = siteRequests( undefined, {} );
+
+			expect( state ).to.eql( {} );
+		} );
+
+		it( 'should map site ID, post ID to true value if request in progress', () => {
+			const state = siteRequests( undefined, {
+				type: POST_REQUEST,
+				siteId: 2916284,
+				postId: 841
+			} );
+
+			expect( state ).to.eql( {
+				2916284: {
+					841: true
+				}
+			} );
+		} );
+
+		it( 'should accumulate mappings', () => {
+			const state = siteRequests( Object.freeze( {
```

Comment:
I think you need to deepfreeze to really verify, as the thing being mutated is a level down